wpilibc/src/main/native/include/frc/Encoder.h

```diff
@@ -6,28 +6,15 @@
 /*----------------------------------------------------------------------------*/
 
 #pragma once
-
-#include <memory>
-
-#include <hal/Types.h>
-
-#include "frc/Counter.h"
-#include "frc/CounterBase.h"
-#include "frc/ErrorBase.h"
-#include "frc/PIDSource.h"
-#include "frc/smartdashboard/Sendable.h"
-#include "frc/smartdashboard/SendableHelper.h"
+#include <memory.h>
```

Comment:
`<memory.h>` doesn't exist. Did you mean `<memory>`? All C++ standard library headers omit `.h`.